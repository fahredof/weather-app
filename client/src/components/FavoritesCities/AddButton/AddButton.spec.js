import React from 'react';
import AddButton from "./AddButton";
import addButton from "./svg/addButton.svg";

describe(`Add button image`, () => {
    const button = shallow(<AddButton/>);

    it(`Рендеринг картинки`, () => {
        expect(button.find("img").prop("src")).toEqual(addButton);
    });

    it(`отображается корректно`, () => {
        expect(shallowToJson(button)).toMatchSnapshot();
    });

});