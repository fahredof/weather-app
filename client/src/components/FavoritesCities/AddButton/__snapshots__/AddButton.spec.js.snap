// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add button image отображается корректно 1`] = `
<div
  id="addButton"
>
  <img
    alt="icon"
    src="addButton.svg"
  />
</div>
`;
